### PR TITLE
New Theme

### PR DESCRIPTION
**Pull request by Holzhaus**
This switches the manual to the ReadTheDocs.org theme with some custom Mixxx CSS changes to better fit our orange-y colorscheme:
2020-02-18-01-14-38_1366x768

### CODE REVIEW

**Comment by daschuer** [comment]
Thank you for taking care.

I have some lose related comments:

Would it be possible to make the version number change-able?

It happens quite often that Google rederects me to an outdated manual.
Can we mark outdated and beta versions somehow prominent?

**Comment by Holzhaus** [comment]
> Would it be possible to make the version number change-able?
> 
> It happens quite often that Google rederects me to an outdated manual.
> Can we mark outdated and beta versions somehow prominent?

Yes. Unfortunately, [`sphinxcontrib-versioning`](https://github.com/sphinx-contrib/sphinxcontrib-versioning) and forks are completely broken for me (and everyone else when looking at the bug tracker) and code is a trainwreck because they do monkey patching everywhere. I'll come up with a custom extension that should be much smaller and more straightforward in a separate PR.

**Comment by rryan** [comment]
Jenkins test this please

**Comment by rryan** [comment]
https://builds.renegadetech.mixxx.org/job/manual-prs/38/console

```
Done.
SSH: Connecting from host [ubuntu]
SSH: Connecting with configuration [downloads.mixxx.org] ...
ERROR: java.io.IOException: Unexpected termination of the channel
Build step 'Send build artifacts over SSH' changed build result to UNSTABLE
Setting status of 5ddddfbb35c53d49e63da8f37453242422e327ba to FAILURE with url https://downloads.mixxx.org/builds/manual/prs/PR87 and message: 'Build finished. '
Using context: Jenkins
Finished: UNSTABLE
```

Maybe transient dropped connection to hostgator.

File: source/conf.py [hunk 1]
```diff
@@ -133,7 +135,7 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "_static/mixxx_logo_color_230px.png"
```
[the remark this answers is not in the document]
Comment (Holzhaus) [reply source/conf.py]: yup, done.

File: source/conf.py [hunk 2]
```diff
@@ -114,7 +116,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'mixxx'
```
[the remark this answers is not in the document]
Comment (Holzhaus) [reply source/conf.py]: yes

**Comment by rryan** [comment]
> Looks great, thanks! I'd like to look at a Jenkins test build before merging just to confirm everything works fine on the build server, but the most recent build failed to upload and I'm not sure why.

http://downloads.mixxx.org/builds/manual/prs/PR87/en/index.html 

Looks good!

**Pull request merged**